dev_guide/cron_jobs.adoc

```diff
@@ -127,14 +129,30 @@ Doing this prevents them from generating unnecessary artifacts.
 [[cleaning-up-after-a-cron-job]]
 == Cleaning Up After a Cron Job
 
-Cron jobs can leave behind artifact resources such as jobs or pods. Check if any remain:
+Cron jobs can leave behind artifact resources such as jobs or pods.  As a user it is important
```

Comment:
s/As a user it is important to configure/Configuring history limits is important, so that...